REPL.cpp

```diff
@@ -140,34 +170,150 @@ bool REPL::ExecuteSwift(std::string line)
     if(ShouldLog(LoggingPriority::Info))
     {
         Log("=========AST Before Modifications==========");
-        src_file->dump();
+        tmp_src_file->dump();
     }
-    AddImportNodes(*src_file, m_modules);
-    //TODO(sasha): Load DLLs
```

Comment:
Was this TODO taken care of?

Reply:
I believe so - it got taken care of in the static block of the JIT constructor.

Reply:
Ok this definitively gets handled 2 PRs from now